src/pages/instances/EditInstance.tsx

```diff
@@ -112,32 +112,37 @@ const EditInstance: FC<Props> = ({ instance }) => {
       // ensure the etag is set (it is missing on the yaml)
       instancePayload.etag = instance.etag;
 
-      void updateInstance(instancePayload, project).then((operation) => {
-        const instanceName = values.name;
-        const instanceLink = instanceLinkFromName({
-          instanceName,
-          project,
+      void updateInstance(instancePayload, project)
+        .then((operation) => {
+          const instanceName = values.name;
+          const instanceLink = instanceLinkFromName({
+            instanceName,
+            project,
+          });
+          eventQueue.set(
+            operation.metadata.id,
+            () => {
+              toastNotify.success(<>Instance {instanceLink} updated.</>);
+              void formik.setValues(getInstanceEditValues(instancePayload));
+            },
+            (msg) =>
+              toastNotify.failure(
+                "Instance update failed.",
+                new Error(msg),
+                instanceLink,
+              ),
+            () => {
+              formik.setSubmitting(false);
+              void queryClient.invalidateQueries({
+                queryKey: [queryKeys.instances],
+              });
+            },
+          );
+        })
+        .catch((e) => {
+          formik.setSubmitting(false);
+          toastNotify.failure("Instance update failed.", e);
```

Comment:
maybe we should include the `instanceLink` here? Since we are using `toastNotify`, user may visit the history of notifications at a later point and might want to navigate to the instance if they have navigated away